### PR TITLE
Change of title and border radius

### PR DESCRIPTION
**Pull request by marcelojw8**
I think this is a much better design

### CODE REVIEW

**Comment by marcelojw8** [comment]
Should have added an Action to this

**Comment by mjwille** [comment]
I will do it as soon as I can

File: client/index.html [hunk 1]
```diff
@@ -7,11 +7,11 @@
 	<link rel="stylesheet" href="./style/style.css">
 </head>
 <body>
-	<h1>Futebol Brasileiro</h1>
+	<h1>Brazilian Football</h1>
```
Comment (LorenzoWolverine) [inline client/index.html:10]: Should be Foot-Ball

**Review by LorenzoWolverine — CHANGES_REQUESTED** [review]
Change "Football" to "Foot-Ball"

File: client/index.html (hunk 1 above)
Comment (marcelojw8) [reply client/index.html:10]: OK. Will change.

**Comment by marcelojw8** [comment]
Made the requested changes.

**Review by LorenzoWolverine — APPROVED** [review]
Oh Yes!

**Pull request merged**